### PR TITLE
[FR] localize `DevSecOps`

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
[FR] localize `DevSecOps`

### Related issue number or link (ex: `resolves #issue-number`)
Solves https://github.com/cncf/glossary/issues/2293

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/devsecops.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title: DevSecOps 
+status: Completed
+category: concept
+tags: ["méthodologie", "sécurité", ""]
+---
+
+## Ce que c'est
+
+Le terme DevSecOps fait référence à une fusion culturelle des responsabilités de développement, opérationnelles et de sécurité.
+Il étend l'approche [DevOps](/fr/devops/) pour inclure les probélmatiques de sécurité
```
Comment (sestegra) [inline content/fr/devsecops.md:11]: Typo "problématiques"

**Review by sestegra — CHANGES_REQUESTED** [review]

File: content/fr/devsecops.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+category: concept
+tags: ["méthodologie", "sécurité", ""]
+---
+
+## Ce que c'est
+
+Le terme DevSecOps fait référence à une fusion culturelle des responsabilités de développement, opérationnelles et de sécurité.
+Il étend l'approche [DevOps](/fr/devops/) pour inclure les problématiques de sécurité
+avec un minimum de perturbations, voire nulle, du flux de travail des développeurs et des opérations.
+Comme DevOps, DevSecOps est un changement culturel, poussé par les technologies adoptées, avec des méthodes d'adoption uniques.
```
Comment (guillaumebernard84) [inline content/fr/devsecops.md:13]: Vous pensez quoi de ces "unique adoption methods" ? ça a un sens selon vous?

Comment (sestegra) [reply content/fr/devsecops.md:13]: Je ne vois pas ce que cela veut dire.

**Review by sestegra — APPROVED** [review]
LGTM 🚀

File: content/fr/devsecops.md (hunk 2 above)
Comment (guillaumebernard84) [reply content/fr/devsecops.md:13]: Si on met "des méthodes d'adoptions particulières", ça aurais plus de sens. Mais est-ce que ça aurais le même sens qu'en anglais? On atteint les limites de mon niveau d'anglais 😢

**Review by Krast76 — APPROVED** [review]

**Pull request merged**